libmamba/src/api/configuration.cpp

```diff
@@ -2048,6 +2048,25 @@ namespace mamba
             fs::u8path(util::user_home_dir()) / ".conda/condarc.d",
             fs::u8path(util::user_home_dir()) / ".condarc",
         };
+        
+        std::array<std::string, 3> condarc_list = {"/.condarc", "/condarc", "/condarc.d"};
+        if (util::get_env("XDG_CONFIG_HOME"))
+        {
+            const std::string xgd_config_home = util::get_env("XDG_CONFIG_HOME").value();
+            for (const auto& path: condarc_list)
+            {
+                conda_user.push_back(fs::u8path(xgd_config_home) + "/conda"  + path);
```

Comment:
I think you can rely on the concatenation operator of the path (ie operator/) instead of that of strings:

`conda_user.push_back(fs::u8path(xgd_config_home) / "conda"  / path);`

This requires to remove the heading `/` in the member of the condarc_list above.